src/lib/vocab/entity/entityServices.ts

```diff
@@ -36,7 +37,11 @@ export const ReadEntitiesService: ServiceByName['ReadEntities'] = {
 		const {hub_id} = await repos.space.findByEntity(source_id);
 		await checkHubAccess(actor, hub_id, repos);
 
-		const pageTies = await repos.tie.filterBySourceIdPaginated(source_id, pageSize, pageKey);
+		const extraPageSize = pageSize ? pageSize + 1 : DEFAULT_PAGE_SIZE + 1;
```

Comment:
just a little clearer: 

```suggestion
		const extraPageSize = (pageSize ?? DEFAULT_PAGE_SIZE) + 1;
```